metrics/cli.go

```diff
@@ -0,0 +1,75 @@
+package metrics
+
+import (
+	"errors"
+	"math"
+
+	"github.com/Layr-Labs/eigenda-proxy/common"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	EnabledFlagName    = "metrics.enabled"
+	ListenAddrFlagName = "metrics.addr"
+	PortFlagName       = "metrics.port"
+	defaultListenAddr  = "0.0.0.0"
+	defaultListenPort  = 7300
+)
+
+var ErrInvalidPort = errors.New("invalid metrics port")
+
+func DefaultCLIConfig() CLIConfig {
+	return CLIConfig{
+		Enabled:    false,
+		ListenAddr: defaultListenAddr,
+		ListenPort: defaultListenPort,
+	}
+}
+
+func CLIFlags(envPrefix string) []cli.Flag {
+	return []cli.Flag{
+		&cli.BoolFlag{
+			Name:    EnabledFlagName,
+			Usage:   "Enable the metrics server",
+			EnvVars: common.PrefixEnvVar("METRICS_ENABLED"),
+		},
+		&cli.StringFlag{
+			Name:    ListenAddrFlagName,
+			Usage:   "Metrics listening address",
+			Value:   defaultListenAddr, // TODO: Switch to 127.0.0.1
```

Comment:
why this todo?

Reply:
[34aa9e9](https://github.com/Layr-Labs/eigenda-proxy/pull/264/commits/34aa9e987203d18765c6a91d6c61e412fd1cdb12)